api/openapi/v2/system-agent.yaml

```diff
@@ -0,0 +1,634 @@
+      type: object
+      properties:
+        requestId:
+          description: "Uniquely identifies the request that resulted in this response."
+          type: string
+          format: uuid
+          example: "e6e8a2f4-eb14-4649-9e2b-175247911369"
+        statusCode:
+          description: "A numeric code signifying the operational status of the response."
+          type: integer
+        message:
+          description: "A field that can contain a free-form message, such as an error message."
+          type: string
+      required:
+        - requestId
+        - statusCode
+    ErrorResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      description: "A response type for returning a generic error to the caller."
+      type: object
+    GetConfigResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      description: "Provides a response containing the configuration for the targeted service."
+      type: object
+      properties:
+        config:
+          description: "A string-ified representation of the service's configuration. For purposes of this specification, a string has been used since configuration structure differs from service to service."
+          type: string
+      required:
+        - config
+    HealthRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      properties:
+        service:
+          description: "The services from which to obtain health information."
+          type: string
+          example: "edgex-core-data"
+    HealthResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      type: object
+      description: "Response containing the health status for the targeted service(s)."
+      properties:
+        service:
+          type: string
+          example: "edgex-core-data"
+        health:
+          type: string
+    MetricsRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      properties:
+        service:
+          description: "The service from which to obtain metrics."
+          type: string
+          example: "edgex-core-data"
+    MetricsResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      type: object
+      properties:
+        executor:
+          description: "The type of executor used to process the retrieve the metrics"
+          type: string
+          example: "docker"
+        operation:
+          description: "The type of operation performed"
+          type: string
+          example: "metrics"
+        service:
+          description: "The service assocaited with the metrics"
+          type: string
+          example: "edgex-core-data"
+        result:
+          description: "The metric information retireved from the targeted service"
+          type: object
+          additionalProperties:
+            type: object
```

Comment:
There is a standard response with a non-standard component.  Include the standard response; you can "object" the non-standard object.

![image](https://user-images.githubusercontent.com/464327/74246387-0d544800-4caa-11ea-965f-a0ed09544297.png)

(`raw` object is non-standard)